help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -0,0 +1,182 @@
+
+### Map attributes and identities {#map}
+
+The next step is mapping source identifiers to the Magnite device_id identifier.
+
+![map desired data fields to the device_id field](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-field-mapping.png)
+
+**Real-Time destination notes**:
+
+- You may have multiple source attributes/namespaces that could represent a Device ID, so map as many as you need.
+
+- A new mapping can be added using the Add new mapping button.
+
+In this example using the Real-Time destination, we're mapping any rows
+that contain a generic deviceId source identifier to the Magnite
+device_id target field. When you\'re done, click Next.
+
+![Be sure to set mapping ids to all activated audiences, or set NONE if no mapping id is present](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-mappingid.png)
+
+You must now configure a Start date (mandatory), End date (optional),
+and a Mapping ID for each audience.
+
+**Mapping ID notes:**
+
+- The Mapping ID field should be used when an audience has a pre-existing Segment ID previously known to Magnite.
+
+- To add a Mapping ID to an audience, click each audience row individually and enter data in the right-hand column (see image above). If you do not want to add a Mapping ID, please enter NONE into the Mapping ID field. This is because the Mapping ID is a mandatory field within the Adobe system, despite technically being optional for a customer.
+
+Once these configurations are applied, click Next.
+
+#### Confirm the destination activation Configuration.
+
+`![](../../assets/catalog/advertising/magnite/destination-realtime-active-audience-review.png)
+
+In this step, confirm the destination activation configuration and click
+Finish.
+
+## Exported data / Validate data export {#exported-data}
+
+Once your audiences have been uploaded, you may validate your audiences
+have been created and uploaded correctly using the following steps:
+
+- The Real-Time destination system will attempt to send the events with a p95 of \<10 mins after a Profile segmentation/Update event. This means that 95% of the time, profile segment updates will be delivered to Magnite Streaming in under 10 min. The actual receipt and processing of the events within Magnite Streaming depends on the shared data volume.
+
+- Post-ingest, segments are expected to appear in Magnite Streaming within a few minutes and can be applied to a deal. You can confirm this by looking up the segment ID that was shared during the activation steps in the Adobe Experience Platform.
+
+  - **Note:** Audiences shared with Magnite Streaming using the  Real-Time destination will also need to be shared using the Magnite Streaming: Batch destination. When configured correctly, segment names in the Magnite Streaming UI are updated to reflect those used in the Adobe Experience Platform post-daily update.
```

Comment:
Let's turn this note into a dedicated heading: `## Activate the same audiences through the Magnite Streaming: Batch destination`, and use that new heading to explain this requirement of using the second destination.

Reply:
Updated